c7n/resources/dynamodb.py

```diff
@@ -666,3 +669,59 @@ def process(self, resources, event=None):
         subnet_groups = client.describe_subnet_groups()['SubnetGroups']
         self.groups = {s['SubnetGroupName']: s for s in subnet_groups}
         return super(DaxSubnetFilter, self).process(resources)
+
+
+@Table.filter_registry.register('consecutive-backups')
+class TableConsecutiveBackups(Filter):
+    """Returns tables where number of consective daily backups is
+    equal to/or greater than n days.
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: dynamodb-daily-backup-count
+                resource: dynamodb-table
+                filters:
+                  - type: consecutive-backups
+                    days: 7
+    """
+    schema = type_schema('consecutive-backups', days={'type': 'number', 'minimum': 1},
+        required=['days'])
+    permissions = ('dynamodb:ListBackups', 'dynamodb:DescribeBackup', 'dynamodb:DescribeTable')
+    annotation = 'c7n:Dynamodb-backups'
+
+    def process_resource_set(self, client, resources):
+        paginator = client.get_paginator('list_backups')
```

Comment:
we should use the `TimeRangeLowerBound` so we dont pull in extraneous backups here since we know the max number of days we want to look back for

Reply:
added the `TimeRangeLowerBound` filter